Sources/EpoxyBars/BarInstaller/BarWrapperView.swift

```diff
@@ -156,29 +156,29 @@ public final class BarWrapperView: UIView {
     }
   }
 
-  private func setModel(_ model: InternalBarCoordinating?, animated: Bool) {
+  private func _setModel(_ originalModel: BarModeling?, animated: Bool) {
```

Comment:
Renaming argument to `originalModel`. It can either be the original `BarModel` type if it has not previously been erased to `AnyBarModel` or can be an `AnyBarModel` type.

Reply:
Sounds good to me